tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/assets/TestClasses/UA0013.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Web.Http;
+using System.Collections.Generic;
+
+namespace TestProject.TestClasses
+{
+    public class ValuesController : ApiController
+    {
+        // GET api/values
+        public IEnumerable<string> GetValues()
+        {
+            return new[] { "value1", "value2" };
+        }
+    }
+
+    public class MoviesController : System.Web.Http.ApiController
+    {
+        // GET api/values
+        public IEnumerable<string> GetValues()
+        {
+            return new[] { "Star Wars", "Iron Man", "Star Trek" };
+        }
+    }
+
+    public class NotAWebController : Foo.ApiController
```

Comment:
Maybe include a `using Foo` at the beginning of this namespace so that the inheritance here can just be `: ApiController` (should work the same, but looks more ambiguous that way).